pkg/provenance.go

```diff
@@ -267,3 +280,60 @@ func verifyProvenanceName(name string) error {
 
 	return nil
 }
+
+// Note: see https://github.com/sigstore/cosign/blob/739947de3d0197fbaab926bd9b896963ebf47a19/pkg/providers/github/github.go.
+func getReusableWorkflowID() (string, error) {
+	url := os.Getenv(requestURLEnvKey) + "&audience=" + audience
+
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return "", err
+	}
+
+	req.Header.Add("Authorization", "bearer "+os.Getenv(requestTokenEnvKey))
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return "", err
+	}
+	defer resp.Body.Close()
+
+	var payload struct {
+		// Count string `json:"count"`
```

Comment:
Drop this or add a comment to docs?
 fwiw i can't find any ref for this json besides that the token is contained in `value`
https://github.com/google-github-actions/auth/blob/48c46e6a594e8bd0b0b4806155ace5cad3aa8e24/src/client/workload_identity_client.ts#L207